### PR TITLE
change 'is false' to '== false'

### PR DESCRIPTION
**Pull request by darrindg91**
Was getting an error: 
FAILED! => {"msg": "The conditional check 'checkmk_server_allow_downgrades | bool is false\nand item.stdout is version(item.item.version, '>')\n' failed. The error was: template error while templating string: no test named 'false'.  so replaced 'is false' to '== false'

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

When trying to upgrade server versions, I see an error:

FAILED! => {"msg": "The conditional check 'checkmk_server_allow_downgrades | bool is false\nand item.stdout is version(item.item.version, '>')\n' failed. The error was: template error while templating string: no test named 'false'. 

## What is the new behavior?

The upgrade now works:

Pausing for 5 seconds
(ctrl+C then 'C' = continue early, ctrl+C then 'A' = abort)
[tribe29.checkmk.server : Short pause for patch update.]
Proceeding with updating site test from version 2.1.0p17.cre to version 2.1.0p20.cre.
This is a minor patch update. A backup will be created in /tmp. This can take a while! The site will be down during the update

-
-
-

### CODE REVIEW

File: roles/server/tasks/update-site.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     msg: |
       Downgrading is not allowed since checkmk_server_allow_downgrades is set to false.
   when:
-    checkmk_server_allow_downgrades | bool is false
```
Comment (msekania) [inline roles/server/tasks/update-site.yml:22]: Actually 
`checkmk_server_allow_downgrades | bool`
is sufficient

Comment (msekania) [reply roles/server/tasks/update-site.yml:22]: please, do not use `== false`

Comment (darrindg91) [reply roles/server/tasks/update-site.yml:22]: Wouldn't that be equivalent to 'is true'? Perhaps:
`not checkmk_server_allow_downgrades`
or something similar?

Comment (robin-tribe29) [reply roles/server/tasks/update-site.yml:22]: Jup, I think that is the best syntax. But do not forget `| bool` at the end.

Comment (msekania) [reply roles/server/tasks/update-site.yml:22]: should read

`not checkmk_server_allow_downgrades | bool`

Comment (darrindg91) [reply roles/server/tasks/update-site.yml:22]: @msekania 

`checkmk_server_allow_downgrades` is defined as false in the defaults of the role, so as long as this is the case this should not be an issue.

Comment (robin-tribe29) [reply roles/server/tasks/update-site.yml:22]: Yes, correct, thanks for catching that @msekania!
@darrindg91 can you update that line to `checkmk_server_allow_downgrades is defined and not checkmk_server_allow_downgrades | bool`? Then we are good to go.

Comment (darrindg91) [reply roles/server/tasks/update-site.yml:22]: @robin-tribe29 

I updated for the not defined case and the corrected 'is false' syntax.

Comment (robin-tribe29) [reply roles/server/tasks/update-site.yml:22]: Great, the logic looks good to me!
If you could fix the linting finding from the tests, we should be good to merge.

**Review by robin-tribe29 — APPROVED** [review]

**Pull request merged**